composer.json

```diff
@@ -8,10 +8,9 @@
     "bevacqua/dragula": "3.7.2",
```

Comment:
Ik had tijdens de upgrades ook bij alle packages gekeken of er upgrades waren, en dat bij de laatste meteen meegenomen. Dat kan er nog wat security issues enzo uithalen, en daar maak je Jeffrey weer blij mee. Kun je eventueel nog doen.

Let wel op dat `doctrine/orm` bijvoorbeeld echt breaking is. Dus dat raad ik niet aan zonder er echt tijd voor te nemen.